basyx.components/basyx.components.docker/basyx.components.AASServer/src/test/java/org/eclipse/basyx/regression/AASServer/MqttAASServerSuite.java

```diff
@@ -123,18 +123,21 @@ public void submodelLifeCycle() throws InterruptedException {
 
 		Submodel submodel = createSubmodel(submodelIdentifier.getId(), submodelIdentifier);
 		manager.createSubmodel(shellIdentifierForSubmodel, submodel);
-
+		System.out.println(listener.getTopics());
```

Comment:
Revert complete class due to irrelevance of println and formatting changes for overall class